README.md

```diff
@@ -6,7 +6,8 @@
 The PrimGrp package provides the library of primitive permutation 
 groups which includes, up to permutation isomorphism (i.e., up to
 conjugacy in the corresponding symmetric group), all primitive 
-permutation groups of degree < 4096.
+permutation groups of degree < 8192. Groups of degree 8096 to 8191
+must be downloaded separately from https://doi.org/10.5281/zenodo.10411367. 
```

Comment:
That's not really enough information for the user to know what to do. However, it seems the website under that URL provides some hints on how to install, so perhaps the text here could state that. Say:
```suggestion
must be downloaded separately from https://doi.org/10.5281/zenodo.10411367
and then installed by following the instructions given there.
```